sympy/vector/coordsysrect.py

```diff
@@ -445,15 +446,17 @@ def lame_coefficients(self):
         return self._lame_coefficients
 
     def transformation_to_parent(self):
-        return self._transformation_lambda(*self.base_scalars())
+        return self._transformation_from_parent_lambda(*self.base_scalars())
```

Comment:
is this a name mismatch? When I suggested to invert the transformation directions, I meant to keep the name consistent here.

Reply:
`self._transformation_from_parent_lambda` should be in `transformation_from_parent`.

Reply:
I also saw this problem. But only in that configuration test will pass

Reply:
Name consistency please.